src/test/java/com/solvd/shop24/AuthenticationTest.java

```diff
@@ -0,0 +1,42 @@
+package com.solvd.shop24;
+
+import com.qaprosoft.carina.core.foundation.utils.ownership.MethodOwner;
+import com.solvd.shop24.gui.pages.shop24.HomePage;
+import com.solvd.shop24.gui.pages.shop24.profile.ProfilePage;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class AuthenticationTest extends BaseTest {
+//1. Page is opened
+//2. Search input field has typed text (for example: "Когтеточка")
+//3. ProductPage is opened (random product item has price and title,
+// which contain typed text). For example:
+//"Когтеточка Trixie Sisal 43071 (бежевый)" contains "Когтеточка"
+
+    @Test(description = "JIRA#AUTO-0008")
+    @MethodOwner(owner = "yantoniuk")
+    public void successAuthentication() {
+        String phoneNumber = "444544281";
+        String password = "cmn0hd";
+        HomePage homePage = openHomePage(getDriver());
+        ProfilePage profilePage = homePage.getMenu().authorization(phoneNumber, password);
+        Assert.assertEquals(getDriver().getCurrentUrl(),
+                "https://24shop.by/cabinet/settings/", "authentication is failed!");
```

Comment:
for url validation please use isPageOpened() or assertPageOpened() methods from carina